pallets/crowdloan/src/lib.rs

```diff
@@ -0,0 +1,246 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2021 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+//! # Crowdloan rewards Pallet
+//!
+//! Provides means of registering the contributors to the KILT crowdloan.
+//!
+//! - [`Config`]
+//! - [`Call`]
+//! - [`Pallet`]
+//!
+//! ## Interface
+//!
+//! ### Dispatchable Functions
+//! - `set_admin_account` - Set the account that is allow to register and delete
+//!   contribution entries into/from this pallet's storage.
+//! - `set_new_contribution` - Add or replace a crowdload contribution, which
+//!   contains the contributor's address and the contributed amount.
+//! - `remove_contribution` - Remove a contribution entry from the pallet
+//!   storage.
+//!
+//! ## Genesis config
+//!
+//! The crowdloan contributions pallet depends on the [`GenesisConfig`].
+//!
```

Comment:
S: Maybe add a comment on what this pallet requires from the genesis config. Similar to the sudo pallet:

> //! You need to set an initial superuser account as the sudo `key`.

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/264/commits/b34620daaa1a9c6144a077843986d4c3bfc4c4b6